templates/sectionsummary.html

```diff
@@ -14,29 +14,63 @@
   })
 }}
 {% block form_content %}
-  {% if content.summary.groups %}
-    {% set group = content.summary.groups %}
-    <h1>{{content.summary.title}}</h1>
-    <div>
-      {% include 'partials/summary/summary.html' %}
-    </div>
-  {% elif content.summary.custom_summary %}
-    <h1>{{content.summary.title}}</h1>
-    {% for summary in content.summary.custom_summary %}
-      {% if summary.type == 'List' %}
+  {% if content.summary.show_non_item_answers == True %}
+    {% if content.summary.title %}
+      <h1>{{content.summary.title}}</h1>
+    {% endif %}
+    {% if content.summary.groups %}
+      {% set group = content.summary.groups %}
+      <div>
+        {% include 'partials/summary/summary.html' %}
+      </div>
+    {% endif %}
+    {% if content.summary.custom_summary %}
+      {% for summary in content.summary.custom_summary %}
+        {% if summary.type == 'List' %}
+          {% set add_link = summary.add_link %}
+          {% set add_link_text = summary.add_link_text %}
+          {% set empty_list_text = summary.empty_list_text %}
+          {% set list_title = summary.title %}
+          {% set related_answers = summary.related_answers %}
+          {% set answer_title = summary.answer_title %}
+
+          <div {{'class="ons-u-mb-l"' if not loop.last }} data-qa="{{summary.list_name}}-list-summary">
+          {% if summary.list %}
+            {% set list = summary.list %}
+            {% include 'partials/summary/list-summary.html' %}
+          {% endif %}
+          </div>
+        {% endif %}
+      {% endfor %}
+    {% endif %}
+  {% else %}
```

Comment:
This won't work because we can't treat lists with related answers as separate to normal answers. With the current logic, list collector answers will always be last. You can have questions after a list collector, so we need to handle list collector as part of the group.  

List collector and its answers should show based on where they show up in the section. If there are 10 questions after the list collector, then those 10 questions need to be displayed after the list collector on the summary. Currently, it looks like it works because there are no questions after the list collector in the test schema.

Reply:
They are ordered correctly now and following the flow of the questionnaire section, thanks!